### DISCUSSION TITLE
Where do you buy OpenMage 20 themes?

### DISCUSSION THREAD

**Discussion by djixas** [OpenMage/magento-lts#3203] [Questions & Answers]
For v19 I could just buy Magento 1.9, but how does one go about running v20 with some non default designs? Themeforest doesn't have anything yet, and I'd love to migrate from v19

**Comment by elidrissidev** [comment]
There is not that many changes in OM 20 that will make it require it's very own themes, so the others should work fine for the most part.

**Comment by addison74** [comment]
If you search in the Discussion section you will find that I have posted more resources. There are no themes for OpenMage, but for Magento 1. You can try Template Monster, but most of them are between 8 and 10 years old. All need changes, updates. I use themes from TM but I removed many extensions that came with the package. OpenMage with a few good extensions can be a solution for a long time to come.

You can also depart from the current RWD theme and create your own. You can use Bootstrap 5.3, Tailwind, AlpineJS. Gone are the days of complex websites because the number of mobile devices has grown rapidly, The desktop is on the decline. A smartphone theme does not need to be a Christmas tree.

**Comment by fballiano** [comment]
IMHO The RWD theme is actually pretty good, you just need a new color scheme (on brand) done be a graphic designer, at the end of the day there's not much to do
